src/nmodl/parse1.ypp

```diff
@@ -1166,7 +1166,7 @@ tablestmt: TABLE tablst dependlst FROM expr TO expr WITH INTEGER
 		  Lappendlst(table_list, $3);
 		  deltokens($1, $9);
 		}
-	| TABLE error { myerr("syntax: TABLE [list] [DEPEND list] FROM expr TO expr WITH integer");}
+	| TABLE error { myerr( "syntax: TABLE [list] [DEPEND list] FROM expr TO expr WITH integer");}
```

Comment:
maybe do a `s/\( "/\("/gc` here :)

Reply:
All are remnants of "too many `char*` casts, take those out". Me taking them out, removing extra stuff ..